trino/client.py

```diff
@@ -75,7 +87,7 @@
 else:
     PROXIES = {}
 
-_HEADER_EXTRA_CREDENTIAL_KEY_REGEX = re.compile(r'^\S[^\s=]*$')
+_HEADER_EXTRA_CREDENTIAL_KEY_REGEX = re.compile(r"^\S[^\s=]*$")
```

Comment:
separate out a commit, having functional and non-functional changes in single commit makes it harder to read history (bisecting for example) in future.

Reply:
Extracted to separate commit